assets/src/apps/authoring/components/RightMenu/PartPropertyEditor.tsx

```diff
@@ -317,6 +317,13 @@ export const PartPropertyEditor: React.FC<Props> = ({
     [currentActivity.id, currentPartInstance, currentPartSelection, dispatch],
   );
 
+  const componentPropertyFocusHandler = useCallback(
```

Comment:
Once a focus is lost on fucus / blur from part property editor, we set the flag to true so that if the user press '`Cntrl+C`' then we trigger the copy part code.